app/src/main/java/org/oppia/android/app/options/LanguageItemViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package org.oppia.android.app.options
+
+import androidx.databinding.ObservableBoolean
+import androidx.lifecycle.LiveData
+import org.oppia.android.app.viewmodel.ObservableViewModel
+
+/**
+ * Language item view model for the recycler view in [AppLanguageFragment] and
+ * [AudioLanguageFragment].
+ */
+class LanguageItemViewModel(
+  val language: String,
+  selectedLanguage: LiveData<String>,
+  val languageRadioButtonListener: LanguageRadioButtonListener
+) : ObservableViewModel() {
+  val isLanguageSelected = ObservableBoolean()
+  init {
+    selectedLanguage.observeForever {
+      isLanguageSelected.set(it == language)
+    }
+  }
+}
```

Comment:
@BenHenning  he approach you've suggested using the `Transformations.map` as in the following gist
<script src="https://gist.github.com/FareesHussain/5c4a36c276f94e9ce55586ea4019e8af.js"></script>
Is not working after lifecycle owner

The issue here is that the radio check is not being updated.

Reply:
I forgot to add lifecyclerOwner for the bindableAdapter. Now it is working.